src/cmd/test.rs

```diff
@@ -36,27 +36,25 @@ pub fn make_subcommand<'help>() -> App<'help> {
             .short('L')
             .long("library-path")
             .value_name("dir")
-            .takes_value(true)
-            .use_delimiter(true)
-            .require_delimiter(true)
-            .multiple_values(true)
-            .multiple_occurrences(true)
-            .forbid_empty_values(true)
+            .value_delimiter(',')
+            .num_args(1..)
             .help("A comma-separated list of directories to add to {n}the crate search path when building tests"))
 }
 
 // test command implementation
 pub fn execute(args: &ArgMatches) -> Result<()> {
-    let library_paths: Vec<&str> = args
-        .values_of("library-path")
-        .map(std::iter::Iterator::collect)
-        .unwrap_or_default();
-    let chapter: Option<&str> = args.value_of("chapter");
+    let library_paths: Vec<&str> = if let Some(it) = args.get_many::<String>("library-path") {
+        it.map(|s| s.as_str()).collect()
+    } else {
+        vec![]
+    };
```

Comment:
I think this can keep the style of the original code by something like:

```suggestion
    let library_paths: Vec<&str> = args
        .get_many("library-path")
        .map(|it| it.map(String::as_str).collect())
        .unwrap_or_default();
```